UI/src/view/municipalitySituationPopup.js

```diff
@@ -0,0 +1,138 @@
+        var latestModificationTableHeaderRow = function () {
+            return '<thead><th id="verifier">K-TUNNUS</th> <th id="date">PVM</th> <th id="name">TIETOLAJI</th></thead>';
+        };
+
+        var latestModificationTableContentRows = function (values) {
+            renameAssets(values);
+            return "<tbody>" +
+                _.map(values, function (asset) {
+                    return '' +
+                        "<tr>" +
+                        "<td headers='verifier'>" + asset.modified_by + "</td>" +
+                        "<td headers='date'>" + asset.modified_date + "</td>" +
+                        "<td headers='name'>" + asset.assetName + "</td>" +
+                        "</tr>";
+                }).join("</tbody>");
+        };
+
+        var tableForGroupingLatestModificationValues = function (values) {
+            return '' +
+                '<table>' +
+                latestModificationTableHeaderRow() +
+                latestModificationTableContentRows(values) +
+                '</table>';
+        };
+
+        return '<div id="dashBoardAssetsModificationsInfo">' + tableForGroupingLatestModificationValues(modificationsInfo) + '</div>';
+    };
+
+    var renderDialog = function(verificationsInfo, modificationsInfo) {
+        $('#work-list').append(me.createMunicipalitySituationPopUp(verificationsInfo, modificationsInfo)).show();
+
+        $('.confirm-modal .cancel').on('click', function() {
+            options.cancelCallback();
+        });
+        $('.confirm-modal .save').on('click', function() {
+            options.saveCallback();
+        });
+
+        $(' .confirm-modal .sulje').on('click', function() {
+            options.closeCallback();
+        });
+    };
+
+    this.createMunicipalitySituationPopUp = function (verificationsInfo, modificationsInfo) {
+        return '' +
+            '<div class="modal-overlay confirm-modal" id="municipalitySituation">' +
+                '<div class="modal-dialog municipalitySituation">' +
+                    '<div class="content">' + options.message + '<a class="header-link sulje"">Sulje</a>' + '</div>' +
+                        '<label class="control-label" id="title">Viimeisimmät päivitykset</label>' +
+                         me.createLatestsAssetModificationsInfoForm(modificationsInfo) +
+                        '<label class="control-label" id="title">Tarkistetut tietolajit</label>' +
+                         me.createCriticalAssetsVerificationInfoForm(verificationsInfo) +
+                    '<div class="actions">' +
+                        '<button class = "btn btn-primary save" onclick="window.location.href=\'#work-list/municipality\'">' + options.saveButton + '</button>' +
+                        '<button class = "btn btn-secondary cancel">' + options.cancelButton + '</button>' +
+                    '</div>' +
+                '</div>' +
+            '</div>';
+    };
```

Comment:
Only if we do a generic Pop Up like this for multiple cases, and in that case, you only will have a generic box because the content will be always different.